app/component-library/components/Pickers/PickerNetwork/PickerNetwork.tsx

```diff
@@ -26,7 +30,14 @@ const PickerNetwork = ({
   const { styles } = useStyles(stylesheet, { style });
 
   return (
-    <TouchableOpacity style={styles.base} onPress={onPress} {...props}>
+    <TouchableOpacity
+      activeOpacity={
+        onPress ? DEFAULT_ACTIVE_OPACITY : WITHOUT_FEEDBACK_ACTIVE_OPACITY
+      }
```

Comment:
Suggestion: if you remove the DEFAULT_ACTIVE_OPACITY const because it's already the default, you can use spread here:
```suggestion
      {...(onPress && { activeOpacity: WITHOUT_FEEDBACK_ACTIVE_OPACITY })},
```

Reply:
Will do it and remove the const created! Thanks!

Reply:
Actually is doing a typescript error, will let this way!